x-team/auto-ui
# Fix Expand / Contract of the "Show" button

**Release Type:** *Non Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-2294
(Tabular view: Expand / Contract of the “Show” button when switching views)

## Description

Now, when switching between List and Tabular view,
the "Show" button will not expand, contract, move,
or show any animation. Basically the button animation
was disabled for this scenario.

Also, making the height of the button consistent
among the two different views

## Checklist

- [x] if any snapshots have been changed, verify that component still works and looks as expected and update changed snapshot

## Related PRs

N/A

## Steps to Test or Reproduce

Switch between List and Tabular view. The "Show" button should not jerk in position or change size with an animation.

## Impacted Areas in Application

* `SearchForm` component in Tabular and List view

## Screenshots

Before: https://www.useloom.com/share/ed18ec4fdf2940cabc1b55fa3c39f598

After:

![xp-2294 b](https://user-images.githubusercontent.com/7514993/46486623-2a325e80-c7cc-11e8-88e1-e64461257bb4.gif)
